clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/ResizeGoogleServerGroupAtomicOperation.groovy

```diff
@@ -63,17 +62,21 @@ class ResizeGoogleServerGroupAtomicOperation implements AtomicOperation<Void> {
     if (isRegional) {
       def instanceGroupManagers = compute.regionInstanceGroupManagers()
 
-      instanceGroupManagers.resize(project,
-                                   region,
-                                   serverGroupName,
-                                   targetSize).execute()
+      timeExecute(
+          instanceGroupManagers.resize(project,
```

Comment:
I think the formatting might look better if it were indented only 2 spaces and the closing parens after `...TAG_REGION, region` were on the next line.
But I don't feel too strongly about it if you prefer it this way.